google/cloud/ndb/context.py

```diff
@@ -300,7 +300,7 @@ def _use_global_cache(self, key, options=None):
             flag = True
         return flag
 
-    def _global_cache_timeout(self, key, options):
+    def _global_cache_timeout(self, key, options=None):
```

Comment:
Maybe this change didn't end up getting used?

Reply:
I just thought that since all the other methods can be called without options, this should work in the same way.

Reply:
My rule of thumb for private methods/functions is only write what is used internally. We can always change it later if need be, but we don't have to anticipate a hypothetical user's use case. We only have to satisfy our own use case.